.github/workflows/clients-dotnet.yml

```diff
@@ -34,17 +34,19 @@ jobs:
           /p:Threshold=\"95,85,95\" \
           /p:ThresholdType=\"line,branch,method\"
       - name: Pack
-        if: startsWith(github.ref, 'refs/tags/')
         working-directory: src/clients/dotnet
         run: |
+          version="$(git tag --points-at ${{ github.sha }})"
+          if [[ "$version" == "" ]]; then
```

Comment:
Instead of using a dummy 0.0.1 version, you can just skip setting `/p:AssemblyVersion=$version /p:Version=$version` in this case. 
We only need to set a version number when publishing to Nuget. For the validation tests, the version number isn't required.

Reply:
Got it, updating, thanks!